### PR TITLE
fixed error that occurred with wstool set when building a workspace w…

### PR DESCRIPTION
**Pull request by markusachtelik**
…here alsam_install and catkin_simple already exist

### CODE REVIEW

**Comment by simonlynen** [comment]
test this please

**Comment by markusachtelik** [comment]
test this please.

**Comment by simonlynen** [comment]
test this please

**Comment by markusachtelik** [comment]
@simonlynen PTAL

File: run_build_impl.sh [hunk 1]
```diff
@@ -137,9 +137,9 @@ then
 	
 	# Make a separate workspace for the deps, so we can exclude them from cppcheck etc.
 	mkdir -p $WORKSPACE/$DEPS
-	cd $WORKSPACE/$DEPS
-	wstool set -t $WORKSPACE/src aslam_install --git git@github.com:ethz-asl/aslam_install.git -y
-	wstool set -t $WORKSPACE/src catkin_simple --git ${CATKIN_SIMPLE_URL} -y
+	cd $WORKSPACE/$DEPS	
+	echo "- git: {local-name: $WORKSPACE/$DEPS/aslam_install, uri: 'git@github.com:ethz-asl/aslam_install.git'}" | wstool  merge -t $WORKSPACE/src -
```
Comment (simonlynen) [inline run_build_impl.sh]: good catch (the workspace).

**Comment by simonlynen** [comment]
lgtm

**Pull request merged**